api/recording_analysis/tasks.py

```diff
@@ -34,23 +32,173 @@ def transcribe_proofread(recording_id, recording_path):
         os.remove(recording_path)
 
         ## Proof read
-        prompt = """The following text is a transcription of an audio recording. 
+        transcription_prompt = """The following text is a transcription of an audio recording. 
                     It may contain errors in interpreted words and punctuation. 
                     Your task is to state the corrected transcription, correcting 
                     the punctuation and aligning the text with what was most likely 
                     intended in the original audio. In your response, state only the
                      correct transcription:""" + transcript.text
+        message_history = [
+            {
+                "role": "user", "content": transcription_prompt
+            }
+        ]
+
+        response = client.chat.completions.create(
+            model="gpt-3.5-turbo",
+            messages=message_history
+        )
+        transcript = response.choices[0].message.content.strip()
+
+        ## Produce entry title
+        message_history.append({"role":"assistant", "content": transcript})
```

Comment:
We might want to consider starting a new message history here instead of appending, since appending keeps the tokens from the conversation so far, which increases cost and processing time. I don't think providing the original transcript gives relevant information that the refined transcript doesn't already have for finding a title.

Reply:
In other words, the new message history should just have the new transcript, correct?

Reply:
Yea that would be better imo